scripts/recovery_sync

```diff
@@ -9,6 +9,12 @@
 
 set -euxo pipefail
 
+function cleanup() {
+	rm -rf "$workdir"
+	rm "${target}.tmp"
```

Comment:
If the file doesn't exist (e.g. after everything runs successfully), will this through an error and cause the script to return an error?